feat: Add reactive/INTRODUCTION_EN.md (#41)
readthedocs-fr/notions: reactive/README.md

# Programmation Réactive 🇫🇷

Ce cours porte sur le paradigme réactif et ses usages.

## Table des matières

- [Introduction](INTRODUCTION_FR.md)

# Reactive Programming 🇪🇳

> This course is concerned about the reactive paradigm and its uses.

Reviewer: ```suggestion
This course focuses on the reactive paradigm and its uses.
```
the former formulation sounded pretty deprecated to me
